flink/v1.12/flink/src/main/java/com/netease/arctic/flink/read/hybrid/assigner/SplitAssigner.java

```diff
@@ -35,8 +37,11 @@ default void open() {
 
   Optional<ArcticSplit> getNext();
```

Comment:
Should we change the result type of this method too?

Reply:
Yes, I return a `Split` instead of `Optional<ArcticSplit>`